src/main/scala/strawman/collection/mutable/TreeSet.scala

```diff
@@ -0,0 +1,193 @@
+package strawman
+package collection.mutable
+
+import collection.{ConstrainedIterableFactory, ConstrainedPolyBuildable}
+import collection.mutable.{RedBlackTree => RB}
+
+import scala.{Boolean, Int, None, Null, NullPointerException, Option, Ordering, Serializable, SerialVersionUID, Some, Unit}
+import java.lang.String
+
+/**
```

Comment:
I didn't check this one thoroughly--I'm assuming it's a reasonably straightforward refactor off of a cut-and-paste.